pkgs/applications/window-managers/wayfire/wcm.nix

```diff
@@ -1,20 +1,56 @@
-{ stdenv, lib, fetchurl, meson, ninja, pkg-config, wayland, wrapGAppsHook
-, gtk3, libevdev, libxml2, wayfire, wayland-protocols, wf-config, wf-shell
+{ stdenv
+, lib
+, fetchFromGitHub
+, meson
+, ninja
+, pkg-config
+, cmake
+, wayland
+, wrapGAppsHook
+, gtk3
+, libevdev
+, libxml2
+, libdrm
+, libinput
+, wayfire
+, wayland-scanner
+, wayland-protocols
+, wf-shell
+, scour
+, libepoxy
 }:
 
 stdenv.mkDerivation rec {
   pname = "wcm";
-  version = "0.7.0";
+  version = "0.7.5";
 
-  src = fetchurl {
-    url = "https://github.com/WayfireWM/wcm/releases/download/v${version}/wcm-${version}.tar.xz";
-    sha256 = "19za1fnlf5hz4n4mxxwqcr5yxp6mga9ah539ifnjnqrgvj19cjlj";
+  src = fetchFromGitHub {
+    owner = "WayfireWM";
+    repo = pname;
+    rev = "v${version}";
+    fetchSubmodules = true;
```

Comment:
Does https://github.com/luispabon/wdisplays not provide a buildable library?

Reply:
Wdisplays is not a library，it should be an optional runtime dependency.
Building wdisplays in wcm is unnecessary